frontend/src/scenes/insights/filters/ActionFilter/entityFilterLogic.ts

```diff
@@ -174,6 +177,13 @@ export const entityFilterLogic = kea<entityFilterLogicType>([
                 index: number
             })
             actions.hideModal()
+
+            await breakpoint(100)
+
+            const dataLogic = insightDataLogic.findMounted({
+                dashboardItemId: props.typeKey,
+            })
+            dataLogic?.actions?.loadData(true)
```

Comment:
This doesn't always fire after setting the renamed series/filters as far as I tested 🤔 
Do you have an idea how to make sure that this happens in the correct order? Not sure I'm using the logics the best way.

Reply:
Also seems to pose a problem in tests... any help appreciated.

Reply:
Most probably `insightDataLogic` with the right prop isn't mounted. No context, but is it supposed to always exist? If yes, you might want to `connect` it so it mounts always with this logic action. I assume then `loadData` will always fire, since it will always exist